### PR TITLE
PAPP-25689: Update the name to "IBM X-Force Exchange"

### PR DESCRIPTION
**Pull request by paulnawat**
### Changes
* Update the name to `IBM X-Force Exchange`
* Once this PR gets merged to `next`, I will also update the name of this repo to `ibmx-forceexchange`. What do you think?
* This is consistent with its name on the API page:
* Asset URL: https://api.xforce.ibmcloud.com/doc/
image

### CODE REVIEW

File: xforce.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
+    "license": "Copyright (c) IBM X-Force Exchange",
     "main_module": "xforce_connector.py",
     "app_version": "1.1.0",
     "utctime_updated": "2022-01-07T20:46:01.000000Z",
     "product_vendor": "IBM",
```
Comment (paulnawat) [inline xforce.json:12]: Should this be updated to `IBM X-Force Exchange` as well?

File: xforce.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-    "name": "IBM XForce",
-    "description": "This app implements various 'investigative' actions on the 'IBM XForce' device",
+    "name": "IBM X-Force Exchange",
+    "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
+    "license": "Copyright (c) IBM X-Force Exchange",
```
Comment (paulnawat) [inline xforce.json:8]: What do you think about this?

File: xforce.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
 {
     "appid": "653a4735-1eed-4cb9-98c4-b40408e60b22",
-    "name": "IBM XForce",
-    "description": "This app implements various 'investigative' actions on the 'IBM XForce' device",
+    "name": "IBM X-Force Exchange",
+    "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
```
Comment (paulnawat) [inline xforce.json:6]: I plan to name the test suite repo to be `ibmx-forceexchange`.
The test playbook will also be named `phantom_ibmx-forceexchange`

File: xforce.json (hunk 1 above)
Comment (joshquerinphantom) [reply xforce.json:12]: The vendor or the connector? Either way, I think currently it's fine. IBM is indeed still the vendor for the product

Comment (paulnawat) [reply xforce.json:12]: @joshquerinphantom  Oops sorry I should have been more clear.
I'm referring to the `product_vendor` here.
Wondering if it should be updated from `IBM` -> `IBM X-Force Exchange`
I think you are right to keep it as `IBM` though. I will let @mattsayar-splunk  confirm

File: xforce.json (hunk 2 above)
Comment (mattsayar-splunk) [reply xforce.json:8]: We're changing the "license" field to "copyright" and we need to keep the current copyright

File: xforce.json (hunk 1 above)
Comment (mattsayar-splunk) [reply xforce.json:12]: Leaving it as IBM is good

File: xforce.json (hunk 3 above)
Comment (mattsayar-splunk) [reply xforce.json:6]: Do all our current test playbooks still use the "phantom_" prefix?

File: xforce.json (hunk 2 above)
Comment (paulnawat) [reply xforce.json]: Thanks @mattsayar-splunk  for reviewing.
I've made the following updates:
* Renamed `license` to `copyright`
* Kept the `Copyright` value the same

File: xforce.json [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
+    "copyright": "Copyright (c) AvantGarde Partners 2017",
     "main_module": "xforce_connector.py",
     "app_version": "1.1.0",
     "utctime_updated": "2022-01-07T20:46:01.000000Z",
     "product_vendor": "IBM",
```
Comment (paulnawat) [inline xforce.json:12]: @mattsayar-splunk I also want to mention that I'm keeping the `product_vendor` as `IBM` here.

File: xforce.json [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
-    "name": "IBM XForce",
-    "description": "This app implements various 'investigative' actions on the 'IBM XForce' device",
+    "name": "IBM X-Force Exchange",
+    "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
+    "copyright": "Copyright (c) AvantGarde Partners 2017",
```
Comment (jdemelo) [inline xforce.json:8]: I think license is a required property in Splunkbase for package validation. At least until BASE-12986 is completed.

Comment (paulnawat) [reply xforce.json:9]: Great catch! I added both `license` and `copyright` fields to make this robust.

**Review by jdemelo — APPROVED** [review]

**Review by alexa-phantom — APPROVED** [review]

**Comment by paulnawat** [comment]
@jdemelo @alexa-phantom @mattsayar-splunk 
Thank you for your help on this PR.
Though there is no error in the pipeline, I’ve just noticed a warning from the install script (see attached).
Upon further checking, I found [a certified app](https://github.com/splunk-soar-connectors/f5bigipltm/blob/next/f5bigipltm.json#L17) with `-` in its name, It looks like we should ignore the `-`.
Therefore, I will change `ibmx-forceexchange` to `ibmxforceexchange`. Please let me know if you have any question or concern. Thanks!

image

**Pull request merged**